ember-flight-icons/package.json

```diff
@@ -106,6 +107,8 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.4.1",
     "eslint-plugin-qunit": "^6.2.0",
+    "husky": ">=6",
+    "lint-staged": ">=10",
```

Comment:
I need to do more investigation but maybe this should only be at root https://www.horacioh.com/writing/setup-lint-staged-on-a-monorepo

If you have any ideas LMK @Dhaulagiri but this is super low priority!